apps/src/aiTutor/views/AITutorFooter.tsx

```diff
@@ -38,125 +32,35 @@ const AITutorFooter: React.FC<AITutorFooterProps> = ({renderAITutor}) => {
   );
   const studentCode = sources[fileMetadata[activeTabKey]].text;
 
-  const hasCompilationError = useAppSelector(
-    state => state.javalabEditor.hasCompilationError
-  );
-  const hasRunOrTestedCode = useAppSelector(
-    state => state.javalab.hasRunOrTestedCode
-  );
-  const isRunning = useAppSelector(state => state.javalab.isRunning);
-  const validationPassed = useAppSelector(
-    state => state.javalab.validationPassed
-  );
-
   const dispatch = useAppDispatch();
 
-  const showCompilationOption =
-    !isRunning && hasRunOrTestedCode && hasCompilationError;
-  const showValidationOption =
-    hasRunOrTestedCode && !hasCompilationError && !validationPassed;
-
   const handleSubmit = useCallback(
-    (actionType: AITutorTypesValue) => {
+    (userMessage: string) => {
       if (isWaitingForChatResponse) {
         return;
       }
 
-      let studentInput = '';
-      let event = '';
-
-      switch (actionType) {
-        case ActionType.COMPILATION:
-          studentInput = QuickActions[ActionType.COMPILATION];
-          event = EVENTS.AI_TUTOR_ASK_ABOUT_COMPILATION;
-          break;
-        case ActionType.VALIDATION:
-          studentInput = QuickActions[ActionType.VALIDATION];
-          event = EVENTS.AI_TUTOR_ASK_ABOUT_VALIDATION;
-          break;
-        case ActionType.GENERAL_CHAT:
-        default:
-          studentInput = userMessage;
-          event = EVENTS.AI_TUTOR_ASK_GENERAL_CHAT;
-          break;
-      }
-
```

Comment:
The chunk of this diff related to compilation and validation is from this PR: https://github.com/code-dot-org/code-dot-org/pull/59238/files#diff-5746b69b8629c76e3f7d3bac321d05897257b3d98f26aecf68d1042707fe3224